Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.yml

```diff
@@ -11,4 +11,4 @@ tags:
   - ml
 timeout: '0'
 type: python
-deprecated: true
\ No newline at end of file
+deprecated: true
```

Comment:
why is this?

Reply:
we put this to all internal scripts that the user should not run.. like the auto mapper or sane doc report